src/noc/containers/noc-fill-form-offline/noc-fill-form-offline.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'exui-noc-fill-form-offline',
+  templateUrl: './noc-fill-form-offline.component.html',
+  styleUrls: ['./noc-fill-form-offline.component.scss']
```

Comment:
This `.scss` file is empty. There's no need for this reference here. And the file should be deleted.

Reply:
file is removed